src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/CountyCourtJudgmentController.java

```diff
@@ -44,10 +43,9 @@ public CountyCourtJudgmentController(
     public Claim save(
         @PathVariable("externalId") String externalId,
         @NotNull @RequestBody @Valid CountyCourtJudgment countyCourtJudgment,
-        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
-        @RequestParam(name = "issue", required = false) boolean issue
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation
     ) {
         String submitterId = userService.getUserDetails(authorisation).getId();
-        return countyCourtJudgmentService.save(submitterId, countyCourtJudgment, externalId, authorisation, issue);
+        return countyCourtJudgmentService.save(submitterId, countyCourtJudgment, externalId, authorisation, false);
```

Comment:
We don'e need `issue` flag to pass in `save` method, because `countyCourtJudgment.ccjType` can be used for the same purpose.